.github/workflows/conformance-aws-cni.yaml

```
@@ -58,7 +58,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}
+  clusterNamePrefix: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}
```

```suggestion
  clusterNamePrefix: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
```

By doing this, continuous retries of the job won't make the clusterNamePrefix change.

---

We want the cluster name to be dedicated per test run as otherwise the test re-run might fail because there's an existing cluster.